docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -160,52 +160,106 @@ This example will produce the following output:
 
 Jobs are created when a task is submitted.
 
-- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`.
+- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`. However, if a job is already being run, it cannot be cancelled. When cancelling a running job, its subsequent jobs will be abandoned and the running job will continue running until completed.
 
-!!! Warning
-    Cancelling a `RUNNING` Job will set the Job to `CANCELLED`. However, it will not terminate the subprocess that has already been dispatched to execute the Job.
+!!! example "Cancelling a job"
 
-!!! Example
+    ```python linenums="1"
+    import taipy as tp
+
+    def double(nb):
+        sleep(5)
+        return nb * 2
+
+    print(f'(1) Number of jobs: {len(tp.get_jobs())}.')
+
+    # Create a scenario then submit it.
+    input_data_node_config = tp.configure_data_node("input", default_value=21)
+    output_data_node_config = tp.configure_data_node("output")
+    double_task_config = tp.configure_task("double_task", double, input_data_node_config, output_data_node_config)
+    print_task_config = tp.configure_task("print_task", print, output_data_node_config)
+    scenario_config = tp.configure_scenario_from_tasks("my_scenario", [double_task_config, print_task_config])
+    scenario = tp.create_scenario(scenario_config)
+    tp.submit(scenario)
+
+    # Retrieve all jobs.
+    print(f'(2) Number of jobs: {len(tp.get_jobs())}.')
+
+    # Get the latest created job of a Task.
+    jobs = tp.get_latest_job(scenario.double_task)
+
+    # Get status of the Job
+    print(f'(3) Status of job double_task: {job[0].status}')
+    print(f'(4) Status of job print_task: {jobs[1].status}')
+
+    # Then cancel it.
+    tp.cancel_job(job[1])
+
+    sleep(10)
+
+    print(f'(5) Status of job double_task: {job[0].status}')
+    print(f'(6) Status of job print_task: {jobs[1].status}')
+    ```
+
+This example will produce the following output:
+
+```
+(1) Number of jobs: 0.
+(2) Number of jobs: 2.
+(3) Status of job double_task: Status.RUNNING
+(4) Status of job print_task: Status.BLOCKED
+(5) Status of job double_task: Status.COMPLETED
+(6) Status of job print_task: Status.CANCELLED
```

Comment:
So I guess this 'CANCELLED' symbol should be... 'CANCELED' right?

Reply:
seems so to me, I'll make another PR to fix that in taipy-core :D